core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -205,6 +208,7 @@ public boolean put(PageId pageId, byte[] page) throws IOException {
         mEvictor.updateOnPut(pageId);
       }
     }
+    LOG.debug("put({},{} bytes) exits with evicting ({})", pageId, page.length, victimPageInfo);
```

Comment:
Shall we differentiate when there is not enough space and the new page fails to add?

Reply:
fixed . good catch